components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.geo-dashboard/geo-dashboard.js

```diff
@@ -17,30 +17,38 @@
  */
 
 function onRequest(context) {
+
     var log = new Log("geo-dashboard.js");
     var devicemgtProps = require("/app/modules/conf-reader/main.js")["conf"];
     var viewModel = {};
-
     var carbonServer = require("carbon").server;
     var device = context.unit.params.device;
-
     var constants = require("/app/modules/constants.js");
-    var wsEndpoint = devicemgtProps["wssURL"].replace("https", "wss") + "/secured-websocket/t/";
-    var spatialWSEndpoint = devicemgtProps["wssURL"].replace("https", "wss");
-    var alertsWSEndpoint = devicemgtProps["wssURL"].replace("https", "wss");
-
+    var wsEndpoint = null;
     var jwtService = carbonServer.osgiService(
         'org.wso2.carbon.identity.jwt.client.extension.service.JWTClientManagerService');
     var jwtClient = jwtService.getJWTClient();
     var encodedClientKeys = session.get(constants["ENCODED_TENANT_BASED_WEB_SOCKET_CLIENT_CREDENTIALS"]);
+    var tokenPair = null;
     var token = "";
     if (encodedClientKeys) {
         var tokenUtil = require("/app/modules/oauth/token-handler-utils.js")["utils"];
         var resp = tokenUtil.decode(encodedClientKeys).split(":");
-        var tokenPair = jwtClient.getAccessToken(resp[0], resp[1], context.user.username, "default", {});
-        if (tokenPair) {
-            token = tokenPair.accessToken;
+        if (context.user.domain == "carbon.super") {
+            tokenPair = jwtClient.getAccessToken(resp[0], resp[1], context.user.username,"default", {});
+            if (tokenPair) {
+                token = tokenPair.accessToken;
+                wsEndpoint = devicemgtProps["wssURL"].replace("https", "wss") + "/secured-websocket/";
+            }
+        } else {
+            tokenPair = jwtClient.getAccessToken(resp[0], resp[1], context.user.username + "@" + context.user.domain,"default", {});
```

Comment:
This line seems exceeding the right margin column which should be 110. Please reformat it.

Reply:
This is Fixed c86c976fe6ca2d0e8b534b7efc441713c7e529b7